packages/fx-core/resource/package.nls.json

```diff
@@ -1017,6 +1017,14 @@
   "configManager.error.driverNotFound": "Driver %s not found for step %s",
   "configManager.error.yamlParsing": "Error parsing yaml file at path %s. Reason: %s",
   "configManager.error.invalidLifecycle": "The schema of lifecycle %s is invalid",
+  "botRegistration.ProgressBar.creatingBotAadApp": "Creating bot aad app.",
+  "botRegistration.log.startCreateBotAadApp": "Creating bot aad app.",
+  "botRegistration.log.successCreateBotAadApp": "Bot aad app created successfully.",
+  "botRegistration.log.skipCreateBotAadApp": "Bot aad app creation skipped.",
```

Comment:
Not sure whether we want aligned wording (with minor changes) with aad app driver, can you sync with Ji on the wording?

Reply:
Good suggestion. Will confirm wording with Ji.